meson.build

```diff
@@ -249,6 +249,10 @@ gcab = find_program('gcab', required : get_option('tests'))
 bashcomp = dependency('bash-completion', required: false)
 python3 = find_program('python3')
 
+if get_option('plugin_uefi_pk')
+  gnutls = dependency('gnutls', version : '>= 3.6.0')
```

Comment:
Do you know if this was the same version needed for libjcat?  If possible we should align the version dependencies if they're skewed.

Reply:
Yes, I chose the same value deliberately.